cmd/snap-repair/runner.go

```diff
@@ -758,22 +783,30 @@ func findBrandAndModel16() (brand, model string, err error) {
 		var err error
 		acct, err = workBS.Get(asserts.AccountType, acctPK, acctMaxSupFormat)
 		if err != nil {
-			return "", "", fmt.Errorf("no brand account assertion in seed data")
+			return nil, fmt.Errorf("no brand account assertion in seed data")
 		}
 	}
 	if err := verifySignatures(acct, workBS, trustedBS); err != nil {
-		return "", "", err
+		return nil, err
 	}
-	return modelAs.BrandID(), modelAs.Model(), nil
+
+	// get the base snap as well
+
+	return &deviceInfo{
+		Brand: modelAs.BrandID(),
+		Model: modelAs.Model(),
+		Base:  modelAs.Base(),
```

Comment:
hmm but on uc16, the model assertion doesn't have a `base` setting, so perhaps we should replace "" with "core" here?

Reply:
yes, I think we probably want to set it to core here

Reply:
I'll set this to core in the followup I think